test/socket.jl

```diff
@@ -239,19 +239,21 @@ begin
     close(sock)
 end
 
-# Local-machine multicast
-
-let
-    function create_socket()
-        s = UDPSocket()
-        bind(s, ip"0.0.0.0", 2000, reuseaddr = true, enable_broadcast = true)
-        s
-    end
-    a, b, c = [create_socket() for i = 1:3]
-    @sync begin
-        recvs = [@async @test bytestring(recv(s)) == "hello" for s in (a, b)]
-        send(c, ip"255.255.255.255", 2000, "hello")
-        map(wait, recvs)
+# Local-machine broadcast
+# (Mac OS X's loopback interface doesn't support broadcasts)
+if @osx ? false : true
```

Comment:
we could move the test to one of the other files that do rely on network access. I'm mostly concerned about the test coverage of ensuring this functionality doesn't regress on osx